src/app/containers/SheriffList.tsx

```diff
@@ -54,7 +54,7 @@ class SheriffList extends React.Component<CompositeProps> {
     }
 
     return (
-      <div style={{ display: 'flex', flexFlow: 'row wrap', justifyContent: 'center' }}>
```

Comment:
I had changed this to justifyContent: 'center' so that when the screen size increased or decreased the row of cards always had even white space on the side.... is it still working like this with 'space-evenly'?